woocommerce/woocommerce-gateway-stripe
# Add UPE

# Changes proposed in this Pull Request:

Adds a new gateway for Stripe UPE.
- Add a feature flag for UPE
- Allow turning UPE on and off, hiding and showing the LPM gateways depending if UPE is enabled, and turning the individual UPE methods on and off.
- Migrate the UPE javascript for classic checkout from normal js asset + minified, to use the wp-scripts pipeline (webpack)

It's composed of the following PRs:
- #1654
- #1647
- #1685

# Testing instructions
- Enable UPE feature flag:
```sql
INSERT INTO `wp_options` (`option_id`, `option_name`, `option_value`, `autoload`) VALUES (NULL, '_wcstripe_feature_upe', '1', 'no') ON DUPLICATE KEY UPDATE `option_value`=VALUES(`option_value`);
```
See each PR (#1654, #1647) specific testing instructions
